pyproject.toml

```diff
@@ -9,6 +9,17 @@ requires = [
 ]
 build-backend = 'setuptools.build_meta'
 
+[tool.cibuildwheel]
+test-requires = "pytest"
+test-command = "pytest -p no:warnings --doctest-rst --pyargs pyflct"
+test-extras = ['dev']
```

Comment:
I think these two need to be specified in the workflow input. The publish template sets these as env vars which will override the values set in pyproject.toml

Reply:
https://github.com/OpenAstronomy/github-actions-workflows/blob/main/.github/workflows/publish.yml#L145-L149

Reply:
SOB

Reply:
Reverted.